### PR TITLE
Sends notification when proxy depositor specified

### LINKED ISSUE #1655: Proxy user designee notification.
### Descriptive summary
Proxy user designee does not get notified after they have been designated as a proxy by another user.
Sufia 7.x HEAD

### Expected behavior
User designated as a proxy should see a message in user notifications informing them they've been designated the proxy for some other user.

### Actual behavior
No notifications for designated user.

### Steps to reproduce the behavior

1. Create two users: first and other.
2. Log in as the first user.
3. Go to dashboard.
4. Designate other user as a proxy under "Manage Proxys" section.
5. Log out.
6. Log in as the other user.
7. Go to dashboard.

### Related work

Tangentially related to #1609

### PR DESCRIPTION
**Pull request by afred**
Fixes #1655 

Sends notification when proxy depositor specified

@projecthydra/sufia-code-reviewers

### CODE REVIEW

**Comment by njaffer** [comment]
@afred: Is it sending email or a notification message? I thought it was only the notification message, in that case the spec test description would not be 'fires off an email to both grantor and grantee'

**Comment by afred** [comment]
@njaffer :+1: thanks.. I'll fix it.

File: app/controllers/concerns/sufia/depositors_controller_behavior.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ def create
         head :ok
       else
         grantor.can_receive_deposits_from << grantee
+        send_proxy_depositor_added_messages(grantor, grantee)
```
Comment (jcoyne) [inline app/controllers/concerns/sufia/depositors_controller_behavior.rb]: Can you make this a subclass of EventJob and call it asynchronously (`perform_later`)? For example: https://github.com/projecthydra/sufia/blob/master/app/controllers/concerns/sufia/users_controller_behavior.rb#L91

**Comment by mjgiarlo** [comment]
Thank you for your work, @afred!

File: app/controllers/concerns/sufia/depositors_controller_behavior.rb (hunk 1 above)
Comment (afred) [inline app/controllers/concerns/sufia/depositors_controller_behavior.rb]: got merged before i read this. New ticket?

Comment (mjgiarlo) [inline app/controllers/concerns/sufia/depositors_controller_behavior.rb]: Nm, I'd say. I don't think we want to conflate user messaging with event jobs. Thanks for the PR!

**Pull request merged**